README.md

```diff
@@ -1,75 +1,120 @@
+- data: path to the folder that contains the files to be uploaded
+- manifest: path to the manifest file
+- name-field: column name in the manifest file that contains file names
+- size-field: column name in the manifest file that contains file sizes
+- md5-field: column name in the manifest file that contains file MD5 checksums
+- retries: number of retries the CLI tool will perform after a failed upload 
+- overwrite: if set to “true”, CLI will upload a file to overwrite the file with same name that already exists in the Data Hub submission S3 bucket. If set to “false”, CLI will not upload a file if a file with the same name exists in the Data Hub submission S3 bucket.
+- dryrun: if set to “true”, CLI will not upload any files to the Data Hub submission S3 bucket. If set to “false”, CLI will upload files to the Data Hub submission S3 bucket.
+
+### Execute upload command
+
+Depends on how Python3 was installed, on some systems you need to use “python” instead of “python3” in following command.
+
+`$ python3 src/uploader.py --config configs/file-upload.yml`
+
+## Upload metadata
+
+### Prepare metadata
+
+Put all metadata (TSV) files to be uploaded in the same folder.
+
+### Prepare configuration file
+- Make a copy of the example config file: “crdc-datahub-cli-uploader/configs/ uploader-metadata-config.example.yml”, give it an appropriate name, in this document we name it “metadata-upload.yml”
+- Open the new config file with a text editor, preferably a code editor like Sublime Text, Notepad++, VSCode, Vim, Emacs etc. Please DO NOT use a word processor like Word or Pages to open the config file.
+- Configurations are in “key: value” format. There must be a space between colon and the value.
+- api-url: keep it unchanged, unless you are using an environment other than Data Hub production environment
+- token: paste in the API token saved in previous steps
+- submission: paste in the submission ID saved in previous steps 
+- type: must be set to “metadata”
+- data: path to the folder that contains the metadata (TSV) files to be uploaded. All files with .txt or .tsv extensions inside the folder will be uploaded.
+- intention: can be set to one of the following values: “New”, “Update”, “Delete”
+- retries: number of retries the CLI tool will perform after a failed upload 
+- overwrite: if set to “true”, CLI will upload a file to overwrite the file with same name that already exists in the Data Hub submission S3 bucket. If set to “false”, CLI will not upload a file if a file with the same name exists in the Data Hub submission S3 bucket.
+- dryrun: if set to “true”, CLI will not upload any files to the Data Hub submission S3 bucket. If set to “false”, CLI will upload files to the Data Hub submission S3 bucket.
+
+### Execute upload command
+
+Depends on how Python3 was installed, on some systems you need to use “python” instead of “python3” in following command.
+
+`$ python3 src/uploader.py --config configs/metadata-upload.yml`
```

Comment:
Windows user use python instead of python3 although only python 3.11 is installed.